### ISSUE TITLE
Discriminator Loss Specification

### ISSUE THREAD

**Issue by cgarciae** [samet-akcay/ganomaly#22]
Amazing work! Just a comment, in the paper its not very clear if to train the discriminator you where going to:

1. Maximize `Ladv` for a (real, fake) pair 
2. Minimize the `cross_entropy` loss for `D(x) = 1` and `D(G(x)) = 0`

In the repo I see you use cross entropy but it would be good to clarify it in the paper, since you don't mention this its hard to understand the _adversarial_ aspect of the training method.

**Comment by samet-akcay** [comment]
Thanks very much for your feedback. I've now updated the repo, which uses feature matching loss for discriminator training.

**Comment by samet-akcay** [comment]
Due to inactivity, I am closing the issue now. Anyone can reopen it for further discussions.

**Comment by robert780612** [comment]
Does your feature matching loss correct?
I guess you want to maximum Ladv loss at model.py:142, but you minimize it.
https://github.com/samet-akcay/ganomaly/blob/master/lib/model.py#L142

**Comment by AlexHex7** [comment]
@samet-akcay @cgarciae I have the same question. Should the loss be multiply by -1?
